linters/queries/get-project-items.graphql

```diff
@@ -0,0 +1,38 @@
+query ($endCursor: String, $login: String!, $project: Int!, $batch: Int!) {
```

Comment:
@widal001 awesome! 

(question) Would this make more sense in the `/.github/` folder since each project has their own linters ATM. Though we still have https://github.com/HHS/simpler-grants-gov/blob/main/.pre-commit-config.yaml hanging out there, so it might be time to clean that up and have a more cohesive linting strategy.

Reply:
Hmmm, good point. Maybe it's worth reframing it as `automations/` instead of `linters/` to avoid confusion with linting on a per-project basis. I could imagine us setting up other forms of automations that aren't strictly linting and that involve automating things outside of GitHub too, which is why I might be hesitant to move all of this over to `.github/`

What are your thoughts on change the directory name and github actions to `automations/` instead of `linters/`?

Reply:
Based on discussion in Slack, we decided to move the `linters/` sub-directory to `.github/linters/`